src/sentry/sentry_metrics/aggregation_option_registry.py

```diff
@@ -22,17 +23,41 @@ class TimeWindow(Enum):
     "d:transactions/measurements.lcp@millisecond": {AggregationOption.HIST: TimeWindow.NINETY_DAYS},
 }
 
-USE_CASE_AGG_OPTION = {UseCaseID.CUSTOM: {AggregationOption.TEN_SECOND: TimeWindow.SEVEN_DAYS}}
+# Currently there are no default per-use case aggregation overrides
+# They are all set via options
+USE_CASE_AGG_OPTION = {}
 
 
-def get_aggregation_options(mri: str) -> dict[AggregationOption, TimeWindow] | None:
+def set_use_case_aggregation_options():
+    # As of now this option is still restricted
+    # to the scope of the custom use case
+    if options.get("sentry-metrics.10s-granularity"):
+        USE_CASE_AGG_OPTION[UseCaseID.CUSTOM] = {
+            AggregationOption.TEN_SECOND: TimeWindow.SEVEN_DAYS
+        }
+
+    for use_case in options.get("sentry-metrics.drop-percentiles.per-use-case"):
+        USE_CASE_AGG_OPTION[UseCaseID(use_case)] = {
+            AggregationOption.DISABLE_PERCENTILES: TimeWindow.NINETY_DAYS
+        }
+
+
+def get_aggregation_options(mri: str, org_id: int) -> dict[AggregationOption, TimeWindow] | None:
+
+    # Set various aggregation options that
+    # are use case-wide aggregations
+    set_use_case_aggregation_options()
+
     use_case_id: UseCaseID = extract_use_case_id(mri)
 
-    # We check first if the particular metric ID has a specified aggregation
-    if mri in METRIC_ID_AGG_OPTION:
-        return METRIC_ID_AGG_OPTION.get(mri)
+    # We check first if the org ID has disabled percentiles
+    if org_id in options.get("sentry-metrics.drop-percentiles.per-org"):
+        return {AggregationOption.DISABLE_PERCENTILES: TimeWindow.NINETY_DAYS}
```

Comment:
There's still an issue in the logic. We need to be able to:
- Disable percentiles for an entire use case
- Override (re-enable) percentiles for a specific org in a specific use case. 

This option will allow us to disable percentiles for specific orgs, but not the inverse due to the fall-through. Ideally, we'd allow for both ways, but if you prefer to keep things simple we can also stick to opt-in for percentiles. What this means is we need two changes:

1. Allow to configure percentiles per usecase on org-level
2. Ensure that the org-level setting _always_ overrides the global one.

Initially, I was considering organization feature flags for this, but given that we better not call out to flagr in the ingest path, sentry options are the better choice.